### ISSUE TITLE
Unable to mapleader to <Space>

### ISSUE THREAD

**Issue by spenserlee** [NeoVintageous/NeoVintageous#469]
### Description
Unable to set the leader key in NeoVintageous to the spacebar key. This is a [fairly](https://stackoverflow.com/questions/446269/can-i-use-space-as-mapleader-in-vim) [common](https://www.reddit.com/r/vim/comments/2d2za5/question_about_having_spacebar_as_leader/) thing to do in regular vim.

### Steps to Reproduce
1. Open `.vintageousrc`
2. Add/edit: `let mapleader=" "`
3. Reload `.vintageousrc` (`ctrl+shfit+p` +  "NeoVintageous: Reload My .vintageousrc File")
4. Execute a command which requires leader.
    * e.g.: `nnoremap <leader>z :ToggleZoomPane<CR>`

### Expected behaviour
Space can be mapped as the leader key.

### Actual behaviour
No command is executed as the leader key appears to be unassigned.

### Versions

Sublime Text 3.2 Build 3200
NeoVintageous 1.10.0 - 2019-03-05
Ubuntu 18.0.4.2 LTS

### Additional Information
Confirmed leader commands work when using the default `let mapleader=,`.

Attempted the following `.vintageousrc` settings:
```
noremap <Space> <Nop>
let mapleader=' '
```
```
let mapleader=" "
```
```
let mapleader="\<Space>"
```

**Comment by gerardroche** [comment]
This should work:

```
let mapleader=<space>
```

But you're right, some other variations currently don't work. For example `let mapleader=<Space>` should work, it's fixed in the upcoming version: #433.

Also, quoting should also work: `let mapleader='<space>'` and `let mapleader="<space>"`. I'll fix this in the next version too.

And literal space should also work too: `let mapleader=" "`. I'll fix this in the next version too.

**Comment by spenserlee** [comment]
> This should work:
>     `let mapleader=<space>`
Of course I didn't try that one :stuck_out_tongue: . Thanks for the quick fix!
